lib/rbUtil.js

```diff
@@ -199,4 +180,11 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+// Store titles as MediaWiki db keys
```

Comment:
In that case I'd suggest a mwUtils.js in lib/, but tbh placing only one function inside seems ... silly. Maybe move it once we have more stuff to put in there?